web/src/views/Login.vue

```diff
@@ -19,13 +19,14 @@
       <div class="flex min-h-48 items-center justify-center bg-wp-primary-200 dark:bg-wp-primary-300 md:w-3/5">
         <WoodpeckerLogo preserveAspectRatio="xMinYMin slice" class="h-32 w-32 md:h-48 md:w-48" />
       </div>
-      <div class="flex min-h-48 flex-col items-center justify-center gap-4 text-center md:w-2/5">
-        <h1 class="text-xl text-wp-text-100">{{ $t('welcome') }}</h1>
+      <div class="flex min-h-48 flex-col items-center justify-center gap-4 p-4 text-center md:w-2/5">
+        <h1 class="-mb-4 text-xl text-wp-text-100">{{ $t('welcome') }}</h1>
```

Comment:
Now we just need to remove this `-mb-4` again 😁

Reply:
Still the think the margin is too big now but don't wanna argue anymore 😄️

Reply:
But why? We have not added any extra space here or what do I miss? The new padding is places on the outer div. The title and the buttons are both direct childs of the  div with the new padding. As such the parents padding does not add space between the childs. Or what do I miss?

Reply:
Sure, there was no vertical padding around the div before. Also, if I add this interactively, I see the space between the header and the button increasing.

Reply:
Alright, Ill give it a test again if Im back on a PC. If this change increases the space between the childs something else is wrong.